examples/lwt/lwt_get_pipelined.ml

```diff
@@ -32,21 +32,11 @@ let request host =
       (Uri.of_string host)
   in
   let* response = Client.get client "/" in
-  let open Lwt_syntax.Async in
   let* () =
-    Lwt_stream.iter_s
-      (fun chunk -> Lwt_io.printf "%s" chunk)
-      (Body.to_string_stream response.body)
+    Body.iter_string_s (fun chunk -> Lwt_io.printf "%s" chunk) response.body
```

Comment:
@anmonteiro `Body.iter_string_s` returns the value of `or_error`, but it looks like we are swallowing the error here. Our main run function checks only for an error resulting from `Client.get client "/blog"`, and appears to ignore errors from `Client.get client "/"`. Is that correct?

Reply:
No, you'll only get there if there are no errors. It's subtle, but the `let*` is `Lwt_result.bind`.

Reply:
Thanks for explaining. I'll have to investigate further where the process is bailing out.

Reply:
It's not bailing out. It's equivalent to this:

```ocaml
let open Lwt_result.Infix in

Body.iter_string_s (fun chunk -> ...) >>= fun () ->
  request "/blog"
...
```

So you're only even issuing the second request if the first succeeded.

Reply:
I see! makes sense